module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -1167,20 +1167,51 @@ public function getMyHolds($patron)
             '/request-storage/requests',
             $query
         ) as $hold) {
-            $requestDate = date_create($hold->requestDate);
+            $requestDate = $this->dateConverter->convertToDisplayDate(
+                "Y-m-d H:i",
+                $hold->requestDate
+            );
             // Set expire date if it was included in the response
             $expireDate = isset($hold->requestExpirationDate)
-                ? date_create($hold->requestExpirationDate) : null;
+                ? $this->dateConverter->convertToDisplayDate(
+                    "Y-m-d H:i",
+                    $hold->requestExpirationDate
+                )
+                : null;
+            //set lastPickup Date if provided, format to j M Y
```

Comment:
Very minor point, but I'd put a space after the `//` here for consistency with other comments.

Reply:
Sorry for oversights like this...